controller/array_action/array_mediator_ds8k.py

```diff
@@ -525,9 +526,10 @@ def _create_flashcopy(self, source_volume_id, target_volume_id, options=None):
                                                                          target_volume_id))
             else:
                 raise ex
-        if not self.validate_flashcopy(api_flashcopy.id):
+        flashcopy_state = self.get_flashcopy_state(api_flashcopy.id)
+        if not flashcopy_state == 'valid':
             self._delete_flashcopy(api_flashcopy.id)
-            logger.info("Flashcopy is not in a valid state")
+            logger.info("Flashcopy state is not correct. expected: 'valid' , got: '{}'.".format(flashcopy_state))
```

Comment:
please move this "info" to the actual raised error message

Reply:
done